spring-batch/src/main/java/org/baeldung/conditionalflow/step/NumberInfoProcessor.java

```diff
@@ -0,0 +1,17 @@
+package org.baeldung.conditionalflow.step;
+
+import org.baeldung.conditionalflow.model.NumberInfo;
+import org.springframework.batch.core.ExitStatus;
+import org.springframework.batch.core.StepExecution;
+import org.springframework.batch.core.annotation.BeforeStep;
+import org.springframework.batch.core.listener.ItemListenerSupport;
+import org.springframework.batch.item.ItemProcessor;
+
+public class NumberInfoProcessor implements ItemProcessor<NumberInfo, Integer> {
+    private StepExecution stepExecution;
```

Comment:
This field seems to be redundant, if so let's remove it, please.

Reply:
Done